Receiving format warnings on JavaScript files

While completing the changes for https://github.com/mdn/browser-compat-data/pull/6087 I received a number of warnings about formatting in JavaScript files. As far as I could tell these files are mainly unchanged since last year, and the last change was made to "Prettier" them.

```
npx: installed 1 in 5.61s
Checking formatting...
scripts/fix-browser-order.js
scripts/fix-feature-order.js
scripts/fix.js
scripts/migrations/002-remove-webview-flags.js
scripts/mirror.js
scripts/release-notes.js
scripts/statistics.js
scripts/traverse.js
test/lint.js
test/linter/test-browsers.js
test/linter/test-consistency.js
test/linter/test-links.js
test/linter/test-prefix.js
test/linter/test-schema.js
test/test-regexes.js
test/test-utils.js
test/utils.js
Code style issues found in the above file(s). Forgot to run Prettier?
/Users/sherpa/Documents/repositories/rebloor/browser-compat-data/test/test-format.js:11
    let errorText = err.stdout.toString();
                               ^

TypeError: Cannot read property 'toString' of null
    at testFormat (/Users/sherpa/Documents/repositories/rebloor/browser-compat-data/test/test-format.js:11:32)
    at Object.<anonymous> (/Users/sherpa/Documents/repositories/rebloor/browser-compat-data/test/lint.js:163:13)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mdn-browser-compat-data@1.0.19 lint: `node test/lint`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mdn-browser-compat-data@1.0.19 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sherpa/.npm/_logs/2020-05-05T14_56_21_548Z-debug.log
```

--- a/.prettierrc
+++ b/.prettierrc
@@ -1,5 +1,6 @@
 {
   "$schema": "http://json.schemastore.org/prettierrc",
+  "arrowParens": "avoid",
   "singleQuote": true,
   "trailingComma": "all"
 }

